src/staging/src/pages/sitemap.js

```diff
@@ -4,21 +4,38 @@ import MegaMenu from "../components/megaMenu.js"
 import { Helmet } from "react-helmet"
 import Header from "../components/header"
 import Footer from "../components/footer"
+import { Link, Breadcrumbs } from "@material-ui/core"
 import "../styles/staticContent1.scss"
 
 
 const SitemapPage = ({ data }) => {
 
   const menuContent = data?.allStrapiMenus?.nodes || []
 
+    const breadcrumbs = [
+    <Link key="1" href="/">
+      Home
+    </Link>,
+    <div key="2" className="breadcrumb-text">
+      Site Map
+    </div>,
+  ]
```

Comment:
nit: I think this is a bit more readable directly nested inside the <Breadcrumbs> component rather than included via array. The array makes me think it's dynamic in some way.